courses/machine_learning/deepdive2/structured/labs/5_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>Remove: This is a verbatim repeat of <a href="https://github.com/gwendolynstripling/training-data-analyst/blob/master/courses/machine_learning/feateng/labs/bqml_adv_feat_eng.ipynb" target="_blank">https://github.com/gwendolynstripling/training-data-analyst/blob/master/courses/machine_learning/feateng/labs/bqml_adv_feat_eng.ipynb</a></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>